src/layers/layer-update.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable guard-for-in */
+
+/**
+ *
+ * @param {Object} updateTriggers {getPosition: {column}, getData: {filteredIndex}}
+ * @param {Object} oldUpdateTriggers
+ * @returns {}
+ */
+export function diffUpdateTriggers(updateTriggers, oldUpdateTriggers = {}) {
+  const triggerChanged = {};
```

Comment:
do you need both variables since at the end you are assigning the value of triggerChanged to reason?
Can't you just return triggerChanges?

Reply:
The return value should be `false` if nothing changed, or `triggerChanged` as an object